apps/ospTutorial/CMakeLists.txt

```diff
@@ -8,14 +8,28 @@ find_package(Boost 1.42.0 REQUIRED COMPONENTS system thread regex)
 include_directories(${Boost_INCLUDE_DIRS} ${PROJECT_SOURCE_DIR})
 link_directories(${Boost_LIBRARY_DIRS})
 
+find_package( Protobuf REQUIRED CONFIG )
+if ( Protobuf_FOUND )
+    message( STATUS "Protobuf version : ${Protobuf_VERSION}" )
+else()
+    message( WARNING "Protobuf package not found -> specify search path via Protobuf_DIR variable")
```

Comment:
afaik you won't get here, since REQUIRED will bail. The whole if is unneeded, imo.

Reply:
yeah to much copy-paste - will fix that